create.js

```diff
@@ -36,9 +37,22 @@ exports.write_3 = function(name, variant, dev) {
 	}
 
 	//Make folder for blank JSON
+	// var newFolderDir = "public/"+name;
+	// if (!fs.existsSync(newFolderDir)){
+	//     fs.mkdirSync(newFolderDir);
+	// }
+
+	// The name you use for the folder creation
 	var newFolderDir = "public/"+name;
```

Comment:
These could probably be moved up with the other constants.

Reply:
Makes sense to me, just moved it. Looks more organized now